Jack/Gen.fs

```diff
@@ -434,6 +434,15 @@ module Gen =
     let float : Gen<float> =
         double |> map float
 
+    let dateTime : Gen<System.DateTime> =
+        gen { let! y = range 1753 9999
```

Comment:
If you're wondering about `1753` and `9999`, it's the date range in SQL Server ([source](http://stackoverflow.com/a/548373/467754)).